rtos/source/TARGET_CORTEX/mbed_lib.json

```diff
@@ -79,12 +79,6 @@
         "MCU_NRF51": {
```

Comment:
Should I also remove NRF51 since they're deprecated now?

Reply:
Check with @MarceloSalazar

Reply:
Oh actually it's fine to remove it, because the option is more related to SoftDevice (which we removed) rather than NRF51 or 52.

Reply:
Done